charts/merlin/Chart.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
     condition: vault.enabled
   - name: mlp
     repository: "https://caraml-dev.github.io/helm-charts"
-    version: 0.2.0
+    version: 0.2.2
```

Comment:
since https://github.com/caraml-dev/helm-charts/pull/53 is merged should it be bumped to 0.2.3 now?

Reply:
can you change this to `~0.2.3` this ensures version range as follows: `0.2.2 <= range < 0.3.0`
can be done for other caraml deps too. 

ref: https://github.com/caraml-dev/helm-charts/pull/47/files#diff-e0965270c1f4719e24a4a8fdb50b31235ceb39821a566ea6b0607d82a9eff804R33